config/qt5_quick.mpb

```diff
@@ -0,0 +1,6 @@
+project: qt5_gui {
```

Comment:
Qt5Qml in its own file and Qt5Quick derived from that? Are the includes necessary, for none of the other qt5 base projects we add includes

Reply:
Please split it in 2 separate files